magento/magento2
# Ability to define Area for dev:di:info CLI command #38758

### Description (*)
PR that adds the ability to specify in `php bin/magento dev:di:info <class> <area>` command the desired area code. 

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#38758

### Manual testing scenarios (*)

1. Run `php bin/magento dev:di:info <class> <area>`
2. In the console you can see information about di for the selected class under a given scope ( standard GLOBAL + defined in the command parameters )

### Questions or comments
Example: 
`app/code/Magento/Backend/etc/adminhtml/di.xml`
![2024-05-26_05-21](https://github.com/magento/magento2/assets/85498741/6f2ec97f-2e7b-47c4-8984-0529372af025)


Note : on the current 2.4. branch, an error will occur when outputting information. It is not related to the current task, see :  https://github.com/magento/magento2/issues/38740

### Contribution checklist (*)
